web-app/src/main/webapp/script.js

```diff
@@ -13,15 +13,36 @@
 // limitations under the License.
 
 let geocoder;
+let map;
 
 function initMap() {
   geocoder = new google.maps.Geocoder();
-  const map = new google.maps.Map(document.getElementById("map"), {
+  map = new google.maps.Map(document.getElementById("map"), {
     center: { lat: -34.397, lng: 150.644 },
     zoom: 5,
   });
   const infoWindow = new google.maps.InfoWindow();
   displayUserLocation(map, infoWindow);
+  fetchMarkers();
```

Comment:
super nit: Can you try removing map global variable by sending map as parameter for fetchMarkers and createMarkerForDisplay?

Reply:
You've done that in your PR though right, should I just leave it for when we merge our brances?